dll/directx/ksuser/ksuser.c

```diff
@@ -195,7 +195,7 @@ KsCreatePin(HANDLE FilterHandle,
             PHANDLE  ConnectionHandle)
 {
     ULONG BufferSize = sizeof(KSPIN_CONNECT);
-    PKSDATAFORMAT DataFormat = ((PKSDATAFORMAT) ((ULONG_PTR)Connect + sizeof(KSPIN_CONNECT)));
+    PKSDATAFORMAT DataFormat = (PKSDATAFORMAT)(Connect + 1);
```

Comment:
I prefer this way of writing it too, but does this really make any difference?

Reply:
It makes a difference for my eyes (bleeding vs not bleeding).